smallrye-reactive-messaging-amqp/revapi.json

```diff
@@ -0,0 +1,55 @@
+[
+    {
+        "extension": "revapi.java",
+        "id": "java",
+        "configuration": {
+            "missing-classes": {
+                "behavior": "report",
+                "ignoreMissingAnnotations": false
+            }
+        }
+    },
+    {
+        "extension": "revapi.filter",
+        "configuration": {
+            "elements": {
+                "include": [
+                    "class io.smallrye.reactive.messaging.amqp.AmqpMessage",
```

Comment:
At some point, it would be interesting to either split connectors into two parts (like Kafka) with an API and impl modules, or clearly defined what public API from what's implementation details.

Reply:
Yes we should at least have a separate package for public API like message and metadata types.